templates/taxi/manufacturer_confirm_delete.html

```diff
@@ -0,0 +1,11 @@
+{% extends "base.html" %}
+
+{% block content %}
+    <h1>You want delete {{ manufacturer }}</h1>
+    <h1>Are you sure to delete this manufacturer?</h1>
+    <h1 style="color: red;">!!!All cars with this manufactured will be also deleted!!!</h1>
```

Comment:
The message 'All cars with this manufactured will be also deleted' contains a grammatical error. It should be 'All cars with this manufacturer will also be deleted.'